readthedocs/proxito/views/hosting.py

```diff
@@ -443,7 +443,7 @@ def _v0(self, project, version, build, filename, url, user):
                         )
                         if filename
                         else None,
-                        "root_selector": "[role=main]",
+                        "root_selector": project.addons.doc_diff_root_selector,
```

Comment:
Are we actually setting `[role=main]` anywhere in this PR as the default? It seems we're losing that default selector, unless the JS is using that when it's `null`?

Reply:
I removed the default from the API because we weren't using it in the JS code --it was hardcoded there.

I put the default value in the JS now, at https://github.com/readthedocs/readthedocs-client/blob/0995c065b45a02b206937ce2bcc6ef0973ab2b10/src/docdiff.js#L82-L83

I'm on the fence about where the default should be, so, I'd appreciate if you have any preference/suggestion.

Reply:
Gotcha. I think the JS is probably fine?